prettier.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   // Custom
   printWidth: 120,
+  trailingComma: "es5"
```

Comment:
nit: ironically, would put a trailing comma here

Reply:
Actually, I've disabled adding trailing comma to keep diff to less files/lines.

The trailing comma added while formatting can't be controlled by config https://prettier.io/blog/2023/07/05/3.0.0.html#print-trailing-comma-in-type-parameters-and-tuple-types-when---trailing-commaes5-14086httpsgithubcomprettierprettierpull14086-14085httpsgithubcomprettierprettierpull14085-by-fiskerhttpsgithubcomfisker